Gibe.Umbraco.V7.Blog/Filters/DateBlogPostFilter.cs

```diff
@@ -4,28 +4,28 @@ namespace Gibe.Umbraco.Blog.Filters
 {
 	public class DateBlogPostFilter : IBlogPostFilter
 	{
-		private readonly int _year;
-		private readonly int? _month;
-		private readonly int? _day;
+		public int Year { get; set; }
+		public int? Month { get; set; }
+		public int? Day { get; set; }
```

Comment:
Was it a deliberate decision to leave the setters public? (For this, and other Filters)

Reply:
It was - I think (assumed) it's needed so it can be deserialized?  If not then I can get rid!

Reply:
Oooh - no...no you do _not_ need a public setter!  I'll remove those offending setters **immediately**!
![image](https://github.com/Gibe/Gibe.Umbraco.V7.Blog/assets/13795483/20316ced-3f06-473f-8966-fabebf338768)

Reply:
![image](https://github.com/Gibe/Gibe.Umbraco.V7.Blog/assets/13795483/e9d255ac-7244-466d-86fe-3831645ae1be)